File: app/src/main/cpp/JNIs/jni_interface.cpp
#include "jni_interface.h"
#include <jni.h>
#include <string>
using namespace acuf;


namespace {
}
JNI_METHOD(jlong, JNIonCreate)(JNIEnv* env, jclass , jobject asset_manager){
//    new assetLoader(AAssetManager_fromJava(env, asset_manager));
//    manager = new Manager;
//    nativeAddr =  getNativeClassAddr(new vrController());
//    setupShaderContents();
//    return nativeAddr;
}
JNI_METHOD(void, JNIonPause)(JNIEnv* env, jclass){

}
JNI_METHOD(void, JNIonDestroy)(JNIEnv* env, jclass){

}
JNI_METHOD(void, JNIonResume)(JNIEnv* env, jclass, jobject, jobject){

}
JNI_METHOD(void, JNIdrawFrame)(JNIEnv*, jclass){

}
JNI_METHOD(void, JNIonGLSurfaceCreated)(JNIEnv * env, jclass){

}
JNI_METHOD(void, JNIonSurfaceChanged)(JNIEnv * env, jclass, jint, jint, jint){

}

File: app/src/main/java/acu/face/JNIInterface.java
package acu.face;

import android.app.Activity;
import android.content.Context;
import android.content.res.AssetManager;

public class JNIInterface {
    private static final String TAG = "JNIInterface";
    static AssetManager assetManager;
    public static native long JNIonCreate(AssetManager asset_manager);
    public static native void JNIonGLSurfaceCreated();
    public static native void JNIonPause();
    public static native void JNIonResume(Context context, Activity activity);
    public static native void JNIonDestroy();
    public static native void JNIonSurfaceChanged(int rotation, int width, int height);
    public static native void JNIdrawFrame();
}
